components/carousel/__examples__/carousel-items.jsx

```diff
@@ -0,0 +1,64 @@
+/* eslint-disable  import/prefer-default-export */
+import React from 'react';
+import Carousel from '~/components/carousel';
+import CarouselItem from '~/components/carousel/private/carousel-item';
+
+export const ITEMS = [
+	{
+		id: 1,
+		heading: 'Carousel Item 1',
+		description: 'Description for #1',
+		src: 'assets/images/carousel/carousel-01.jpg',
+		CTALabel: 'Get Started',
```

Comment:
The `CTALabel` isn't a key we agreed upon. You can keep it in the items array, but only use it the custom `onRenderItem` callback.

Reply:
going with `buttonLabel`, if present a button is rendered with that text, otherwise it is not rendered at all to adhere with the SLDS standard

Reply:
done